test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -1944,8 +1944,10 @@ public void BinaryClassifierFieldAwareFactorizationMachineTest()
 
             // see https://github.com/dotnet/machinelearning/issues/404
             // in Linux, the clang sqrt() results vary highly from the ones in mac and Windows. 
+            // goign for 3 digits of precision, because the range of search is  (-0.0001 - 0.0001)
+            // for one of the values, and the actual value is 0.00099999999999989
```

Comment:
0.00099999999999989 is close enough to 0.001; it looks like the difference is smaller than 10^-7. Why do we need a larger tolerance?

Reply:
Yeah, but it is expecting a number in the -0.0001 to 0.0001 range.  Still looking into as the test artifacts are not saved. 



---
In reply to: [224266472](https://github.com/dotnet/machinelearning/pull/1206#discussion_r224266472) [](ancestors = 224266472)